terraform/main.tf

```diff
@@ -1,15 +1,62 @@
-﻿// declarations for the resources to be created 
+// declarations for the resources to be created 
 // this does not include the state S3 bucket as this must be created in order for terraform to init
 
+locals {
+  postgres_config = defaults(var.postgres_config, {
+    allocated_storage           = 100
+    engine                      = "postgres"
+    engine_version              = "14"
+    port                        = 5432
+    backup_retention_period     = 5

```

Comment:
We should make longer. Maybe 15 - 30 days?

Reply:
Was not sure about the number. I know gov paas performs weekly backups. I'll put 15 in there then.